agent/app/agent_capability.go

```diff
@@ -197,6 +201,28 @@ func (agent *ecsAgent) appendTaskENICapabilities(capabilities []*ecs.Attribute)
 	return capabilities
 }
 
+func (agent *ecsAgent) appendVolumeDriverCapabilities(capabilities []*ecs.Attribute) []*ecs.Attribute {
+	// for non-standardized plugins, call docker pkg's plugins.Scan()
+	nonStandardizedPlugins, err := agent.scanPlugins()
+	if err  == nil {
```

Comment:
both here and a bit lower in the file, can you make sure that error gets logged when set?

Reply:
scanPlugins() already logs when it fails to scan the directory so I didn't do so here. It won't hurt to log again here though. Will add logging.